### PR TITLE
move image-box styling from wikirate to decko (card)

### PR DESCRIPTION
**Pull request by ethn**

### CODE REVIEW

File: card/db/migrate_core_cards/20190823220018_cleanup_for_decko_1_0.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ def up
       delete_card [perm, :right, :options]
     end
     delete_card "*missing link"
-    delete_card "tags"
+    # delete_card "tags"
```
Comment (ethn) [inline card/db/migrate_core_cards/20190823220018_cleanup_for_decko_1_0.rb]: WikiRate has 8000+ `+tag` cards.  We don't want tag in the seed data, but I'm not sure we want to migrate it out of every site.

File: card/mod/bootstrap/set/abstract/bootswatch_theme/thumbnail.haml [hunk 2]
```diff
@@ -1,10 +0,0 @@
-.card.mb-4.box-shadow
```
Comment (ethn) [inline card/mod/bootstrap/set/abstract/bootswatch_theme/thumbnail.haml]: there is no thumbnail_image view in decko now.  I think this haml file must have been obviated?

File: card/mod/media/lib/stylesheets/style_image_box.scss [hunk 3]
```diff
@@ -0,0 +1,79 @@
+//images
```
Comment (ethn) [inline card/mod/media/lib/stylesheets/style_image_box.scss]: from wikirate

File: card/db/migrate_core_cards/20190823220018_cleanup_for_decko_1_0.rb (hunk 1 above)
Comment (xithan) [reply card/db/migrate_core_cards/20190823220018_cleanup_for_decko_1_0.rb]: right, we don't want that

**Pull request merged**